### PR TITLE
Add Lassie's HOL4 tutorial and the Lassie lib to the release notes

### PR DESCRIPTION
**Pull request by HeikoBecker**

### CODE REVIEW

File: Manual/Lassie-Tutorial/intro.tex [hunk 1]
```diff
@@ -0,0 +1,99 @@
+
+\paragraph{Note for CPP 2021 reviewers}
```
Comment (arolle) [inline Manual/Lassie-Tutorial/intro.tex:2]: You could omit this paragraph.

Comment (HeikoBecker) [reply Manual/Lassie-Tutorial/intro.tex]: Thanks! That somehow slipped through.

**Comment by mn200** [comment]
Thanks!

**Comment by mn200** [comment]
Building the manual fails because a `setup_hol4.sh` file is not present.

**Comment by HeikoBecker** [comment]
Sorry, I forgot to add the file.

Thank you for fixing it already!

**Pull request merged**